spec/duration.html

```diff
@@ -1732,12 +1732,14 @@ <h1>
           1. If _zonedRelativeTo_ is not *undefined*, then
             1. Let _intermediate_ be ? MoveRelativeZonedDateTime(_zonedRelativeTo_, _years_, _months_, _weeks_, _days_).
             1. Let _result_ be ? NanosecondsToDays(_nanoseconds_, _intermediate_).
-            1. Set _days_ to _days_ + _result_.[[Days]] + _result_.[[Nanoseconds]] / _result_.[[DayLength]].
+            1. Let _fractionalDays_ be _days_ + _result_.[[Days]] + _result_.[[Nanoseconds]] / _result_.[[DayLength]].
```

Comment:
Any reason why the changes in this commit aren't reflected in polyfill changes?

Reply:
The polyfill uses a different calculation here, since _fractionalDays_ is a mathematical value that might not be representable by a JS Number. I'll try to have the integer math changes resolve this inconsistency.